libs/designsystem/src/lib/scss/_global-styles.scss

```
@@ -129,17 +129,17 @@ ion-modal.kirby-overlay {
           --min-height: #{$modal-default-height};
           align-items: flex-start;
 
-          &.small {
+          &.kirby-modal--small {
```

Please align with existing naming:
```suggestion
          &.kirby-modal-small {
```